spec/models/invoice_spec.rb

```diff
@@ -10,5 +10,29 @@
     it {should belong_to :customer}
     it {should have_many :invoice_items}
     it {should have_many :transactions}
+    it {should have_many(:items).through(:invoice_items)}
+  end
+
+  before do
+    @customer_1 = Customer.create!(first_name: "First", last_name: "Customer")
+
+    @merchant_1 = create(:merchant)
+    @items = create_list(:item, 5, merchant: @merchant_1)
+
+    @invoice_1 = create(:invoice, customer: @customer_1)
+    @invoice_2 = create(:invoice, customer: @customer_1)
+    @invoice_3 = create(:invoice, customer: @customer_1)
+    @invoice_4 = create(:invoice, customer: @customer_1)
+
+    @invoice_items_1 = create(:invoice_item, invoice: @invoice_1)
+    @invoice_items_2 = create(:invoice_item, invoice: @invoice_2)
+    @invoice_items_3 = create(:invoice_item, status: 1, invoice: @invoice_3)
+    @invoice_items_4 = create(:invoice_item, status: 2, invoice: @invoice_4)
+  end
+
+  describe "class methods" do
+    it "incomplete invoices" do
```

Comment:
Could we be a bit more descriptive here, because I am looking at "invoices", when it's actually wanting "incomplete_invoice_items"